test/JDBC/expected/pivot-vu-verify.out

```diff
@@ -2486,3 +2486,25 @@ varchar#!#int#!#int#!#int#!#int#!#int
 OrderNumbers#!#19#!#19#!#19#!#16#!#14
 ~~END~~
 
+
+-- BABEL-5603
+SELECT extra_column, [0], [1], [2], [3], [4]
```

Comment:
Can you add a test case for create a view with this select pivot in prepare, and verify this view's output?